components/gfx/display_list/mod.rs

```diff
@@ -214,10 +214,12 @@ impl DisplayList {
         // We don't perform this adjustment on the root stacking context because
         // the DOM-side code has already translated the point for us (e.g. in
         // `Window::hit_test_query()`) by now.
-        if !is_fixed && stacking_context.id != StackingContextId::root() {
-            if let Some(scroll_offset) = scroll_offsets.get(&stacking_context.id) {
-                translated_point.x -= Au::from_f32_px(scroll_offset.x);
-                translated_point.y -= Au::from_f32_px(scroll_offset.y);
+        if stacking_context.id != StackingContextId::root() {
```

Comment:
Okay. Thanks for pointing that out! I've fixed it in the latest version of the branch.